XRTK-Core/Packages/com.xrtk.core/Providers/Controllers/Hands/HandUtils.cs

```diff
@@ -0,0 +1,137 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System.Collections.Generic;
+using System.Linq;
+using UnityEngine;
+using XRTK.Definitions.Utilities;
+
+namespace XRTK.Providers.Controllers.Hands
+{
+    public static class HandUtils
```

Comment:
Consider changing this to a hand extension rather than a util

Reply:
This whole class feels weird. It does math to reason about joint rotations, but only magic leap doesn't provide those.

Reply:
I'm also not a fan of just adding `Utils` to stuff lol. `Utility` or `Utilities` is just fine. Really no point in abbreviating things.

Reply:
Agreed, definitely no abbreviations :D